src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -1305,6 +1312,13 @@ public virtual Object[] ArgumentList
                    ParameterSetName = InvokeCommandCommand.FilePathVMNameParameterSet)]
         public virtual String ConfigurationName { get; set; }
 
+        /// <summary>
+        /// This parameter specifies the SSH subsystem to use for the remote connection.
+        /// </summary>
+        [Parameter(ValueFromPipelineByPropertyName = true,
```

Comment:
This parameter definition should go in the PSRemotingBaseCmdlet, but this appears to be in the PSExecutionCmdlet class (which derives from PSRemotingBaseCmdlet).  Please move it there under the SSHHostParameters section.

Reply:
This should be the only Subsystem parameter, other cmdlets should be inheriting from this one now.